oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/MeterServiceHandler.java

```diff
@@ -71,11 +71,11 @@ public MeterServiceHandler(ModuleManager manager, KafkaFetcherConfig config) {
 
     @Override
     public void handle(final ConsumerRecord<String, Bytes> record) {
-        try (HistogramMetrics.Timer timer = histogramBatch.createTimer()) {
+        try (HistogramMetrics.Timer ignored = histogramBatch.createTimer()) {
```

Comment:
`timer` to `ignored`? What is the new naming style?

Reply:
> `timer` to `ignored`? What is the new naming style?

IDE raises warnings for unused variables, unless they're `ignored`, `unused`